rusk-recovery/src/state/snapshot/acl.rs

```diff
@@ -0,0 +1,21 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+use dusk_bls12_381_sign::PublicKey;
+use serde_derive::{Deserialize, Serialize};
+
+use super::*;
```

Comment:
While this is perfectly working, it's something we should avoid, unless you're in the very same file (e.g. unit test).
The reason is, from watching this file I have literally no idea what I'm importing here. Plus, I might import stuff that I'm not using – but I use in the super module.